changelog.md

```diff
@@ -22,6 +22,7 @@
 
 ### Changes
 
+- [#3346](https://github.com/ignite/cli/issues/3346) Improve scaffold query --help
```

Comment:
```suggestion
- [#3346](https://github.com/ignite/cli/issues/3346) Improve `scaffold query --help`
```

Reply:
We have been skipping changelog for changes related to docs to make it more readable.